### ISSUE TITLE
version info not being embedded properly

### ISSUE THREAD

**Issue by satra** [dandi/dandi-cli#49]
`pip install dandi` does the following.

```
...
  Created wheel for dandi: filename=dandi-0.0.0-cp37-none-any.whl size=67033 sha256=46af8b37a25b16b497236d7bd4aee6163b5d6fcf01da6f290bc85df90b9de8c9
  Stored in directory: /Users/satra/Library/Caches/pip/wheels/60/a6/58/d841466d5d3849c392a32d656c10daa16affd4d0cc2a0a5bdc
Successfully built dandi
Installing collected packages: tqdm, appdirs, joblib, dandi
Successfully installed appdirs-1.4.3 dandi-0.0.0 joblib-0.14.1 tqdm-4.43.0
```

**Comment by yarikoptic** [comment]
For 0.4.1 I have produced wheel locally and uploaded:
```
(git)lena:~/proj/dandi/dandi-cli-clone[tags/0.4.1^0]git
$> PYTHON=python make -f ../dandi-cli/Makefile release-pypi
# better safe than sorry
test ! -e dist
python setup.py sdist
running sdist
...
removing build/bdist.linux-x86_64/wheel
twine upload dist/*
Uploading distributions to https://upload.pypi.org/legacy/
Uploading dandi-0.4.1-py2.py3-none-any.whl
100%|███████████████████████████████████████████████████████████████████████| 85.0k/85.0k [00:01<00:00, 48.9kB/s]
Uploading dandi-0.4.1.tar.gz
100%|███████████████████████████████████████████████████████████████████████| 89.7k/89.7k [00:01<00:00, 62.6kB/s]

View at:
https://pypi.org/project/dandi/0.4.1/
```
Let's see what this one would say to you ;-)

**Comment by satra** [comment]
works now.

**Comment by satra** [comment]
btw, are we supporting py2? the wheel seems to think so.

**Comment by yarikoptic** [comment]
we don't ... copied a bug from datalad ;)  should not use `--universal`. 
"Fixed" in 21b8505d8aedbdb2f764a727ca7a98e5c36da31e . uploaded dandi-0.4.1-py3-none-any.whl , deleted that py2.py3 one
